src/commands/build.ts

```diff
@@ -142,9 +142,14 @@ async function writeTypes (distDir: string, meta: ModuleMeta) {
     schemaShims.push('  interface NuxtHooks extends ModuleHooks {}')
   }
   if (hasTypeExport('RuntimeModuleHooks')) {
+    consola.warn('Please use \'ModuleRuntimeHooks\' instead of \'RuntimeModuleHooks\' as this misspelling will be removed in the future.')
     moduleImports.push('RuntimeModuleHooks')
     appShims.push('  interface RuntimeNuxtHooks extends RuntimeModuleHooks {}')
   }
+  if (hasTypeExport('ModuleRuntimeHooks')) {
```

Comment:
Should we throw an error when `RuntimeModuleHooks` also present?

Reply:
After testing locally the types seem to work fine with two extends as seen below 🤷‍♂️ it feels like it shouldn't.. Might test it again in the morning just to be sure 😅

```ts
declare module '#app' {
  interface RuntimeNuxtHooks extends RuntimeModuleHooks {}
  interface RuntimeNuxtHooks extends ModuleRuntimeHooks {}
}
```

Reply:
I've changed it so that if multiple interfaces are used it's extended like so:

```ts
declare module '#app' {
  interface RuntimeNuxtHooks extends RuntimeModuleHooks, ModuleRuntimeHooks {}
}
```